Favourite Places/HomeTableViewController.swift

```diff
@@ -0,0 +1,117 @@
+//
+//  HomeTableViewController.swift
+//  Favourite Places
+//
+//  Created by IMCS on 8/11/19.
+//  Copyright © 2019 IMCS. All rights reserved.
+//
+
+import UIKit
+import MapKit
+
+class HomeTableViewController: UITableViewController {
+
+    var annotationArray2 = [MKPointAnnotation]()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        print("*********")
```

Comment:
No print statements in final code, plz remove any print statements, commented code.